epigraphhub/data/worldbank.py

```diff
@@ -0,0 +1,257 @@
+
+
+def get_worldbank_data(ind, country, db=2, time="all", columns=None):
+
+    """
+    This function get a list of indicators according to some country from the world data bank
+    and return this series in a dataframe.
+
+    :params ind: List of strings. List with the indicators whose data you want to get.
+
+    :params country: List of strings|string. List with the ISO-CODE for the countries whose
+                     data you want to get.
+
+    :params time: Interval of  years. If filled `time = 'all'`, the function will
+                  return all the data available. You can also specify a range of years.
+                  For example, if you want to get the data for the period between the
+                  years 2010 and 2020, you can fill this parameter with
+                  `time = range(2010,2021)`.
+
+    :params columns: List of strings. List with names to rename the columns instead of use the
+                    names in the `ind` list.
+
+    Important: The lists `ind` and `columns` must have the same lenght.
+
+    :returns: DataFrame
+
+    """
+
+    if columns != None:
+
+        if len(columns) == len(ind):
+
+            rename_columns = {}
+
+            for i in np.arange(0, len(ind)):
+                rename_columns[ind[i]] = columns[i]
+
+    else:
+
+        rename_columns = {}
+
+    if len(ind) == 1:
+
+        df = wb.data.DataFrame(
+            series=ind, economy=country, db=db, index=["time"], labels=True, time=time
+        )
+
+        df = df.reset_index()
+
+        del df["time"]
+
+        df["Time"] = df["Time"].astype("str")
+
+    else:
+        df = wb.data.DataFrame(
+            series=ind,
+            economy=country,
+            db=db,
+            index=["time", "economy"],
+            labels=True,
+            time=time,
+        )
+
+    if len(rename_columns) != 0:
+
+        df = df.rename(columns=rename_columns)
+
+    new_dates = []
+    periodicity = []
+
+    for i in df["Time"]:
+
+        if len(i) == 4:
+
+            new_dates.append(f"{i}-01-01")
+            periodicity.append("yearly")
+
+        else:
+
+            if i[4] == "M":
+
+                new_dates.append(f"{i[:4]}-{i[-2:]}-01")
+                periodicity.append("monthly")
+
+            if i[4] == "Q":
+
+                dict_month = {
+                    "Q1": "01-01",
+                    "Q2": "04-01",
+                    "Q3": "07-01",
+                    "Q4": "10-01",
+                }
+
+                new_dates.append(f"{i[:4]}-{dict_month[i[-2:]]}")
+                periodicity.append("quarterly")
+
+    df["dates"] = new_dates
+    df["frequency"] = periodicity
+
+    df.set_index("dates", inplace=True)
+
+    df.index = pd.to_datetime(df.index)
+
+    if len(ind) == 1:
+
+        df_new = pd.DataFrame()
+
+        for i in country:
+
+            df_aux = pd.DataFrame()
+
+            df_aux[ind[0].lower().replace(".", "_")] = df[i]
+
+            df_aux["country"] = [i] * len(df_aux)
```

Comment:
If you simply write
```python
df_aux["country"] = i
```
all elements of this column will be filled with i. there is not need to create a list first.